mkosi.clangd

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+set -eux
+
+PROFILE="$1"
+shift
+
+if [[ -n "${MKOSI_CONFIG-}" ]]; then
+    exec mkosi-chroot env --chdir="$SRCDIR/$PROFILE" clangd "$@"
+fi
+
+# If there's a mkosi.conf or mkosi.local.conf in the current working
+# directory, assume that Include=mkosi-kernel is used to include the
+# mkosi-kernel mkosi configuration. Otherwise, assume the user is
+# running mkosi directly from the mkosi-kernel repository.
+if [[ -f mkosi.conf ]] || [[ -f mkosi.local.conf ]]; then
```

Comment:
For better correctness this should also check for `mkosi.local` and `mkosi.conf.d` (though the current two should usually be sufficient)

Reply:
Yeah let's just do these two until there's an actual use case that only uses the directory ones